script.js

```diff
@@ -0,0 +1,658 @@
+// video poker is a single player poker
```

Comment:
consider splitting content of index.js between a few files so that your code is tidier and easier to read